### PR TITLE
Simplified the template as it is meant to be only a template

### PR DESCRIPTION
**Pull request by ldimaggi**
Simplified the template as it is meant to be only a template and not a
best practices guide to writing tests. A best practices guide could be
a document that references multiple existing tests each of which
illustrates a best practice - such as avoiding the use of sleep statements
or the user of xpath to locate UI elements.

### CODE REVIEW

**Comment by ldimaggi** [comment]
@sanketpathak 
Please review - thanks!

File: frontend/integration-tests/views/devconsole-view/template.view.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+//element finder with class name
+export const untilLoadingBoxLoaded = until.presenceOf($('.loading-box__loaded'));
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+
+export const isLoaded = () => browser.wait(until.and(untilNoLoadersPresent, untilLoadingBoxLoaded)).then(() => browser.sleep(1000));
+
+export const resourceRows = $$('[data-test-rows="resource-row"]');
+// export const resourceRowNamesAndNs = $$('.co-m-resource-icon + a');
+
+export const rowForName = (name: string) => resourceRows.filter((row) => row.$$('.co-m-resource-icon + a').first().getText().then(text => text === name)).first();
```
Comment (spadgett) [inline frontend/integration-tests/views/devconsole-view/template.view.ts]: This is definitely *not* a best practice and leads to `StaleElementReferenceError` during tests. We shouldn't encourage this pattern.

File: frontend/integration-tests/views/devconsole-view/template.view.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+import { appHost, testName, waitForNone } from '../../protractor.conf';
+import { resourceTitle } from '../../views/crud.view';
+
+
+export const reloadBtn = $('#reload-object');
+
+//element finder with class name
+export const untilLoadingBoxLoaded = until.presenceOf($('.loading-box__loaded'));
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
```
Comment (spadgett) [inline frontend/integration-tests/views/devconsole-view/template.view.ts]: We should be encouraging the use of `data-test-id` for new tests in place of class selectors.

**Comment by ldimaggi** [comment]
Two things - first, the view file should have been deleted before I created the PR - second - I like Sam's idea of using existing tests as examples/best practices rather than a mostly empty template. Closing this PR in favor of using well written tests as examples/best practices. Thx!

**Pull request closed without merging**